hooks/hookexecution/executor.go

```diff
@@ -166,6 +166,9 @@ func (e *hookExecutor) ExecuteProcessedAuctionStage(request *openrtb_ext.Request
 	outcome.Entity = entityAuctionRequest
 	outcome.Stage = stageName
 
+	// Reset the requestwrapper as bidrequest is modified by hook
```

Comment:
Hooks may not modify the Bid Request. The infrastructure is designed such that any change a Hook wants to make must be expressed as a Change Set Mutation. The bug here seems to be the Mutation functions are given the Bid Request as an argument, but should be given the Bid Request Wrapper.